src/__tests__/components/AddDoubleFields/AddDoubleFields.test.js

```diff
@@ -1,52 +1,50 @@
-import React from 'react';
-import { createStore } from 'redux';
-import { mount } from 'enzyme';
-import toJSON from 'enzyme-to-json';
+/* eslint-env jest */
+import React from 'react'
+import { createStore } from 'redux'
 
-import createTestForm from '../../../utils/createTestForm';
-import { wrap } from '../../../utils/createTestForm';
-import AddDoubleFields from '../../../components/AddDoubleFields/AddDoubleFields';
+import { wrap } from '../../../utils/createTestForm'
+import AddDoubleFields from '../../../components/AddDoubleFields/AddDoubleFields'
 
 const initialValues = {
-    'mock-name': [
-        {
-            name: 'me',
-        },
-        {
-            name: 'you',
-        },
-    ]
-};
```

Comment:
🔥 `;` you are useless baby ^_^

Reply:
I am not so sure :)
This and `no trailing comma` are only two things which I don't like in **standardjs** project :)